groovy/postInit/mod/ICBM.groovy

```diff
@@ -1,691 +1,678 @@
-import classes.*;
+import classes. * ;
```

Comment:
did you just did a replaceAll there

Reply:
yeah

Reply:
should better re-format this using IntelliJ